src/components/__test__/LoggedHeader.test.js

```diff
@@ -0,0 +1,37 @@
+import LoggedHeader from '../LoggedHeader.vue';
+import { shallowMount, createLocalVue } from '@vue/test-utils';
+import Buefy from 'buefy';
+import mock from '../../test/mockBack';
+
+describe('LoggedHeader', () => {
+  let localVue;
+  const mock = { newsLetter: "Hi, I'm a new bar" };
+  beforeEach(() => {
+    const Vue = createLocalVue();
+    Vue.use(Buefy);
+    localVue = Vue;
+  });
+  test('render component', () => {
+    const wrapper = shallowMount(LoggedHeader, {
+      localVue,
+      data() {
+        return { mock };
```

Comment:
here the mock is used as i see

Reply:
Here the object is a return on function, not an object in object